packages/web/components/table/asset-balances.tsx

```diff
@@ -143,11 +146,27 @@ export const AssetBalancesTable: FunctionComponent<{
       },
     }
   );
+
+  const [hideDust, setHideDust] = useState(false);
+
   const assetsData = useMemo(
     () => assetPagesData?.pages.flatMap((page) => page?.items) ?? [],
     [assetPagesData]
   );
-  const noSearchResults = Boolean(searchQuery) && !assetsData.length;
+
+  const filteredAssetsData = useMemo(() => {
+    return assetsData
+      .map((asset) => {
+        const isDust = asset?.usdValue?.toDec()?.lte(DUST_THRESHOLD);
```

Comment:
Thoughts on passing this to the server? Otherwise, doing this on the client could cause there to be empty pages and various sized pages. Overall it may lead to an inconsistent infinite scroll UX.

It could just be a param called minUsdValue or something